### PR TITLE
Add AQA test results publish

### PR DESCRIPTION
**Pull request by sophia-guo**
Add AQA test results publish checklist See https://github.com/adoptium/temurin-build/pull/3901

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 1]
```diff
@@ -158,6 +158,7 @@ Release Week Checklist:
 - [ ] **Check for presence of jdk8u aarch32 GA tag and mirror it** [Upstream Git repo](https://github.com/openjdk/aarch32-port-jdk8u) - [Mirror job](https://ci.adoptium.net/view/git-mirrors/job/git-mirrors/job/adoptium/job/git-skara-aarch32-jdk8u/)
 - [ ] **Do all of the above for the jdk8u/aarch32 build: Ensure to specify overridePublishName param**
 - [ ] **Archive/upload all TCK results**
+- [ ] **Archive/upload all AQA results** [Publish AQA test results](https://github.com/adoptium/temurin-build/blob/master/RELEASING.md)
```
Comment (smlambert) [inline .github/ISSUE_TEMPLATE/release-checklist.md:161]: Too bad we can not hyperlink directly to an anchor at that page more specific than https://github.com/adoptium/temurin-build/blob/master/RELEASING.md#after-build-pipeline-finished as someone then has to search for the info.

**Review by smlambert — APPROVED** [review]
Approving it, thanks @sophia-guo!  

I added a comment about my pet peeve on documentation, where I like to link to anchors, but some docs do not have many anchors for each unique topic in them, so consider it a rant, and not expecting you to resolve it in this PR :)

File: .github/ISSUE_TEMPLATE/release-checklist.md (hunk 1 above)
Comment (sophia-guo) [reply .github/ISSUE_TEMPLATE/release-checklist.md:161]: Yes, I was thinking to refer by code https://github.com/adoptium/temurin-build/blob/master/RELEASING.md?plain=1#L304, but feels not good either.

Comment (sxa) [inline .github/ISSUE_TEMPLATE/release-checklist.md:161]: > Too bad we can not hyperlink directly to an anchor at that page

Yeah that's a fair comment. We could maybe say something like this to make it a little easier but I'm happy to approve regardless :-)
```suggestion
- [ ] **Archive/upload all AQA results** Search for `Publish AQA test results` in [RELEASING.md](https://github.com/adoptium/temurin-build/blob/master/RELEASING.md) for the process.
```

**Pull request merged**